chia/plotting/plot_tools.py

```diff
@@ -316,3 +316,41 @@ def find_duplicate_plot_IDs(all_filenames=None) -> None:
         for filename_str in duplicate_filenames:
             log_message += "\t" + filename_str + "\n"
         log.warning(f"{log_message}")
+
+
+def load_plot(
+    filename: str,
```

Comment:
I think making this a `Path` would be more appropriate. It looks like you also meant it to be `Optional[]`